docs/api.md

```diff
@@ -234,8 +234,70 @@ Result:
 ```
 
 > #### Notes
+>
+> - Clearing the local collection will mark the collection as never synchronized;
 > - Detailed API documentation for `Collection#clear()` is available [here](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-clear).
 
+## Authentication
+
+Authenticating against a Kinto server can be achieved in [three distinct ways](http://kinto.readthedocs.org/en/latest/api/cliquet/authentication.html):
+
+- Using Basic Auth
+- Using an OAuth Bearer Token
+- Using Firefox Account
```

Comment:
Firefox Account and OAuth Bearer Token is basically the same thing.
Also note that it is possible to plug any pyramid auth backend with kinto